style-guide.md

```diff
@@ -85,15 +85,25 @@ Użyj tej sekcji, jako spisu tłumaczeń dla często używanych, technicznych po
 
 | Słowo / pojęcie  | Tłumaczenie |
 | ------ | ----------- |
+| Array | Array (odmieniany po polsku) |
 | Development environment | Środowisko programistyczne   |
 | Development server | Serwer deweloperski   |
+| Explorer | Eksplorator   |
 | Gatsby Docs | Dokumentacja Gatsby (pisane z wielkiej litery)   |
+| Global styles | Style globalne |
+| Layout components | Komponenty Układu |
+| Node | Node (odmieniane po polsku)  |
 | Plugin | Wtyczka     |
 | Query  | Zapytanie   |
+| Root | Katalog główny   |
 | Run command | Uruchom komendę   |
-| String | Łańcuch znaków   |
-| Stylesheet | Arkusz styli   |
+| Scope | Zasięg |
+| Selectors | Selektory |
+| Source plugins | Wtyczki Źródeł   |
+| String | String (odmieniany po polsku)   |
+| Stylesheet | Arkusz stylów   |
 | Theme  | Motyw       |
+| Transformer plugins | Wtyczki Transformacji   |
```

Comment:
Co powiesz na "Wtyczki przekształcające"? W swoich tłumaczeniach trzymałem się tego. Najwyżej można korzystać z tego zamiennie: 
```suggestion
| Transformer plugins | Wtyczki Transformacji / Wtyczki Przekształcające  |
```

Reply:
Podchodzac do tego czysto technicznie, uwazam ze polski wyraz Transformacja jest praktycznie zapozyczeniem slowa Transform, dlatego nie widze lepszego kandytata w tym zestawieniu. Dla mnie też brzmi lepiej, ale to moze kwestia gustu akurat. Uwazam ze powinniśmy się trzymać jednego nazewnictwa, szczegolnie w takich przypadkach gdy opisujemy jeden z kluczowych elementow Gatsby

Reply:
Ok! Mi pasuje.